pom.xml

```diff
@@ -577,7 +577,7 @@
         <version.org.testcontainers>1.16.0</version.org.testcontainers>
         <version.org.testng>7.4.0</version.org.testng>
         <version.org.wildfly.arquillian>5.0.0.Alpha5</version.org.wildfly.arquillian>
-        <version.org.wildfly.core>19.0.0.Beta17</version.org.wildfly.core>
+        <version.org.wildfly.core>19.0.0.Beta18-SNAPSHOT</version.org.wildfly.core>
```

Comment:
Would you mind removing the wildfly-core upgrade from this PR? It is not part of it

Reply:
Done.